### DISCUSSION TITLE
Unable to access APIs

### DISCUSSION THREAD

**Discussion by anand-kulk** [directus/directus#3174] [Archive]
Hello there -
   I have followed the installation instructions and directus is installed and works fine but I am not able to access APIs.
I have environment - Postgres 11, NodeJS 10.23.0, Ubuntu 18.04.

When I hit below APIs it works 
http://server:port/server/info
http://server:port/server/ping

but anything related to project I created doesn't work, looks like API config is not set for my project.
GET /server/projects returns 
{"errors":[{"message":"Route /server/projects doesn't exist.","extensions":{"code":"ROUTE_NOT_FOUND"}}]}

GET /system/info
{"errors":[{"message":"Route //system/info doesn't exist.","extensions":{"code":"ROUTE_NOT_FOUND"}}]}

Any idea?

Regards,
Anand

**Comment by rijkvanzanten** [comment]
Hi @anand-kulk! I think you've fallen victim to our transition period.. We're currently moving everything (website/docs/cloud/youtube/etc/etc/etc/) over from v8 to v9. It sounds like you're applying the v8 docs / endpoints to your v9 installation.

The route would be `/server/info` in v9 👍

**Reply by anand-kulk** [reply]
@rijkvanzanten thanks for clarifying. /server/info works for me, but anything else which is in the context of a project doesn't work. Even the authentication request returns ROUTE_NOT FOUND. I am supplying the project name in API url that I used during npx create project command. Could you guide further? Do you have any V9 docs that I can refer to?

Below endpoints don't work, ex -
/server/projects
/server/{project name}/auth/authenticate
/server/{project name}/*

Appreciate your help.

**Reply by rijkvanzanten** [reply]
v9 no longer has that server scope, other than that the endpoints are mostly the same as in v8:

`/server/info`
`/auth/login`
`/items/articles`

etc

**Reply by anand-kulk** [reply]
@rijkvanzanten Sorry for confusion but by /server/ I meant the host:port where it is deployed and not server as a word. I am able to authenticate now BUT still not able to get any data within my collection. It keeps saying FORBIDDEN You don't have permission to access the collection. GET on /collections, /files etc return empty data response even though there are collections and files uploaded. I am calling API as an administrator. Any idea what could be going wrong?

**Reply by Mgldvd** [reply]
@anand-kulk maybe this will help you:
https://next.docs.directus.io/reference/api/items.html
I also spent about 2 days trying to access the endpoints of version 9 :smile:

**Reply by anand-kulk** [reply]
Hey @Mayccoll , Thanks for pointing me in right direction, I did look at that but when I try to hit anything under items then it complains that I do not have permissions to access the collection. I am an administrator and not sure why would that be.
When I looked in the backend DB(Postgres), I found that in directus_permissions table , the permissions to different collections are only given to Content Editor role and not for the admin role, is that the problem? Not sure, if code is bypassing looking perms for admin role and that's why they are missing...

{
    "errors": [
        {
            "message": "You don't have permission to access the \"xyz\" collection.",
            "extensions": {
                "code": "FORBIDDEN"
            }
        }
    ]
}

Copying @rijkvanzanten

**Reply by anand-kulk** [reply]
@Mayccoll did you get error above when accessing v9 APIs or any idea why I may be getting it?

**Reply by Mgldvd** [reply]
@anand-kulk  no idea.

**Reply by SviatoslavBaranov** [reply]
If you don't have permissions to some collection (probably the name of DB) you can open your Directus profile (it must be privileged user or Admin), open settings >> Access Policies >> Public  ---  there you choose the name of your DB, Directus files, etc. and give a permissions for read/write/share/delete 

<img width="1117" alt="Screenshot 2025-06-12 at 09 40 26" src="https://github.com/user-attachments/assets/90493b3d-b853-443b-a6d3-340372abae9c" />
